src/pages/workspace/invoices/WorkspaceInvoicesFirstSection.js

```diff
@@ -26,8 +26,13 @@ const WorkspaceInvoicesFirstSection = ({translate, policyID}) => (
         icon={MoneyEnvelopeBlue}
         menuItems={[
             {
+<<<<<<< HEAD
                 title: translate('workspace.common.invoices'),
                 onPress: () => openOldDotLink('reports?param={"createInvoice":true}'),
+=======
+                title: translate('workspace.invoices.sendInvoice'),
+                onPress: () => openSignedInLink('reports?param={"createInvoice":true}'),
```

Comment:
`openSignedInLink` is not available in this CP, we will have to CP another PR.

Reply:
Actually it looks like `openSignedInLink` was removed, so I changed this back to use `openOldDotLink`